src/main/groovy/com/netflix/nebula/lint/plugin/GradleLintExtension.groovy

```diff
@@ -37,7 +37,7 @@ class GradleLintExtension {
     boolean alwaysRun = true
     boolean autoLintAfterFailure = true
 
-    List<String> skipForTasks = ['help', 'tasks', 'dependencies', 'dependencyInsight', 'components', 'model', 'projects', 'properties', 'wrapper']
+    List<String> skipForTasks = ['help', 'tasks', 'dependencies', 'dependencyInsight', 'dI', 'components', 'model', 'projects', 'properties', 'wrapper']
```

Comment:
We are using some techniques to match shorthands for task names. Could we use it here to avoid specifying `dI`? It would also not cover `depIn` or other flavors.

Reply:
yeah, I changed it to use task instances vs startParameter.taskNames. it should be good now and updated the test that checks for short hand names https://github.com/nebula-plugins/gradle-lint-plugin/blob/d080b3f318e4922aa302c3446cbecf187f03b948/src/test/groovy/com/netflix/nebula/lint/plugin/GradleLintPluginSpec.groovy#L919